task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sbom-json-check
+spec:
+  description: >-
+    Check the syntax of the sbom-cyclonedx.json file which should be found
+    in the /root/buildinfo/content_manifests/ directory
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name to be verified
+      type: string
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  workspaces:
+    - name: workspace
+  steps:
+  - name: sbom-json-check
+    image: $(params.IMAGE_URL)
+    workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+    securityContext:
+      runAsUser: 0
+      capabilities:
+        add:
+          - SETFCAP
+    script: |
+      cd /tmp
+      curl -OL https://github.com/CycloneDX/cyclonedx-cli/releases/download/latest/cyclonedx-linux-x64
+      dnf -y install libicu jq 2> fail_result.txt
```

Comment:
no way, to install anything in the task. How do you know that curl or dnf are part of the image?

Reply:
Good question. I also feel they might be missing in image.

So I wanted to test by running `./hack/test-builds.sh hacbs`, but it doesn't work because https://github.com/redhat-appstudio/build-definitions/blob/main/hack/test-build/kustomization.yaml#L5 doesn't work for the new folder `task`. After I change the folder to `task`, new issue comes out.

@Michkov Do you know the right way to make `./hack/test-builds.sh hacbs` working?

Reply:
Updating scripts to work with new format.

I would create sidecar from params.IMAGE_URL which would have `cp -r /root/buildinfo/content_manifests/ /shared` where shared would be shared volume used by main script which would be run on image which is including cyclonedx

Reply:
Wow.. Good idea! I thought to extract that folder from image file but feel it is not good...

Reply:
@Michkov Updated. Thank you for the good idea!